app/utils/ERSUtil.scala

```diff
@@ -17,12 +17,11 @@
 package utils
 
 import java.util.concurrent.TimeUnit
-
 import config.{ApplicationConfig, ERSShortLivedCache, ERSShortLivedHttpCache}
+
 import javax.inject.{Inject, Singleton}
 import metrics.Metrics
-import models.{ErsMetaData, ErsSummary, GroupSchemeInfo, ReportableEvents, SchemeOrganiserDetails, TrusteeDetailsList}
-import models.{AltAmendsActivity, AlterationAmends, CheckFileType, CompanyDetailsList}
+import models.{AltAmendsActivity, AlterationAmends, CheckFileType, CompanyDetails, CompanyDetailsList, ErsMetaData, ErsSummary, GroupSchemeInfo, ReportableEvents, SchemeOrganiserDetails, TrusteeDetails, TrusteeDetailsList}
```

Comment:
at this point I think this could be just an `import models._`?

Reply:
Definitely!